maintenance/wikia/sendWeeklyDigest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * sendWeeklyDigest
+ *
+ * This script sends the weekly digest to the users found in the global_watchlist table found
+ * in the dataware database. For more information, see the README found in the GlobalWatchlist
+ * extension. Additional logging can be found on the Weekly Digest dashboard:
+ * https://kibana.wikia-inc.com/#/dashboard/elasticsearch/Weekly%20Digest
+ *
+ * @author james@wikia-inc.com
+ * @ingroup Maintenance
+ */
+
+ini_set( 'display_errors', 'stderr' );
+ini_set( 'error_reporting', E_ALL );
+
+require_once( dirname( __FILE__ ) . '/../Maintenance.php' );
+
+/**
+ * Class sendWeeklyDigest
+ */
+class sendWeeklyDigest extends Maintenance {
+
+	private $dryRun = false;
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Send the weekly digest";
+		$this->addOption( 'dryRun', 'Do a test run of the script without actually sending the digest', false, false, 'd' );
+	}
+
+	public function execute() {
+		$this->dryRun = $this->hasOption( "dryRun" );
+
+		$this->logRunTime();
+		$userIDs = $this->getUserIDs();
+		$this->sendDigestToUsers( $userIDs );
+	}
+
+	private function getUserIDs() {
```

Comment:
This logic can be moved to `GlobalWatchlistTask`. If later needed somewhere else, it can be reused